platform/migration/platform-migratable/src/main/java/org/codice/ddf/migration/impl/PlatformMigratable.java

```diff
@@ -0,0 +1,216 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class handles the export process for all Platform configurations and system files.
+ *
+ */
+public class PlatformMigratable implements Migratable {
+
+    private static final Logger LOGGER = LoggerFactory
+            .getLogger(PlatformMigratable.class);
+
+    private static final String KEYSTORE_SYSTEM_PROP = "javax.net.ssl.keyStore";
+
+    private static final String TRUSTSTORE_SYSTEM_PROP = "javax.net.ssl.trustStore";
+
+    private static final String WS_SECURITY_DIR = "etc/ws-security";
+
+    private static final String SYSTEM_PROPERTIES = "etc/system.properties";
+
+    private static final String USERS_PROPERTIES = "etc/users.properties";
+    
+    private static final String ABSOLUTE_PATH_WARNING =
+            "The value for property [%s] is set to a path [%s] that is absolute; "
+                    + "therefore, the file will not be included in the export.  "
+                    + "Check that the file exists on the system you're migrating to "
+                    + "or update the property value and export again.";
+
+    private static final String OUTSIDE_PATH_WARNING =
+            "The value for property [%s] is set to a path [%s] that is outside [%s]; "
+                    + "therefore, the file will not be included in the export.  "
+                    + "Check that the file exists on the system you're migrating to "
+                    + "or update the property value and export again.";
+
+    private static final String UNREAL_PATH_WARNING =
+            "The value for property [%s] is set to a path [%s] that could not coerced into a real path; "
+                    + "therefore, the file will not be included in the export.  "
+                    + "Check that the file exists on the system you're migrating to "
+                    + "or update the property value and export again.";
+
+    private final Path ddfHome;
+    
+    public PlatformMigratable(@NotNull Path ddfHome) {
+        notNull(ddfHome, "ddfHome cannot be null");
+        this.ddfHome = ddfHome;
+    }
+    
+    @Override
+    @NotNull
+    public MigrationMetadata export(@NotNull Path exportPath) throws MigrationException {
+        notNull(exportPath, "exportPath cannot be null");
+        exportSystemFiles(exportPath);
+
+        Collection<MigrationWarning> migrationWarnings = new ArrayList<>();
+        migrationWarnings.addAll(exportSecurity(exportPath));
+        return new MigrationMetadata(migrationWarnings);
+    }
+
+    @Override
+    @NotNull
+    public String getDescription() {
+        return "Exports all Platform configurations and system files";
+    }
+
+    @Override
+    public boolean isOptional() {
+        return false;
+    }
+    
+    private Collection<MigrationWarning> exportSecurity(Path exportDirectory) {
+        exportDirectory(exportDirectory, WS_SECURITY_DIR);
+
+        Collection<MigrationWarning> migrationWarnings = new ArrayList<>();
```

Comment:
Do we really need the extra collection here? Can't we just `return exportKeystores()`?